src/ORM/DataExtension.php

```diff
@@ -60,42 +63,112 @@ public function augmentWrite(&$manipulation)
     {
     }
 
+    /**
+     * Extend the owner's onBeforeWrite() logic
+     *
+     * See {@link DataObject::onBeforeWrite()} for context.
+     */
     public function onBeforeWrite()
     {
     }
 
+    /**
+     * Extend the owner's onAfterWrite() logic
+     *
+     * See {@link DataObject::onAfterWrite()} for context.
+     */
     public function onAfterWrite()
     {
     }
 
+    /**
+     * Extend the owner's onBeforeDelete() logic
+     *
+     * See {@link DataObject::onBeforeDelete()} for context.
+     */
     public function onBeforeDelete()
     {
     }
 
+    /**
+     * Extend the owner's onAfterDelete() logic
+     *
+     * See {@link DataObject::onAfterDelete()} for context.
+     */
     public function onAfterDelete()
     {
     }
 
+    /**
+     * Extend the owner's requireDefaultRecords() logic
+     *
+     * See {@link DataObject::requireDefaultRecords()} for context.
+     */
     public function requireDefaultRecords()
     {
     }
 
+    /**
+     * Extend the owner's populateDefaults() logic
+     *
+     * See {@link DataObject::populateDefaults()} for context.
+     */
     public function populateDefaults()
     {
     }
 
+    /**
+     * Influence the owner's can() permission check value to be disallowed (false),
+     * or allowed (true) if no other processed results are to disallow.
+     *
+     * See {@link DataObject::can()} and {@link DataObject::extendedCan()} for context.
```

Comment:
You can also use null to opt out of giving an opinion - it might be worth noting that, also applies to the other can* methods

Reply:
I like "opt-out", but went with "open" (so as not to imply the outcome was already opinionated).